src/markdown.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react/addons";
+import mdast from "mdast";
+import mdastReact from "mdast-react";
+
+import CodePane from "./code-pane";
+import Heading from "./heading";
+import Image from "./image";
+import Link from "./link";
+import List from "./list";
+import ListItem from "./list-item";
+import Quote from "./quote";
+import S from "./s";
+import Text from "./text";
+
+// We can't pass props into mdast-react directly, so we have to "bind" them
+// to spectacle components (ex. headings, strong/em/del)
+function spectacleComponent(component, boundProps = {}) {
+  return React.createClass({
+    render(){
+      const props = {...this.props, ...boundProps};
+      return React.createElement(component, {...props}, this.props.children);
+    }
+  });
+}
+
+// Would like to expose this config to the user
+const mdastConfig = {
+  commonmark: true,
+  mdastReactComponents: {
+    a: Link,
+    code: CodePane,
+    del: spectacleComponent(S, {type: "strikethrough"}),
+    em: spectacleComponent(S, {type: "italic"}),
+    h1: spectacleComponent(Heading, {size: 1}),
+    h2: spectacleComponent(Heading, {size: 2}),
+    h3: spectacleComponent(Heading, {size: 3}),
+    h4: spectacleComponent(Heading, {size: 4}),
+    h5: spectacleComponent(Heading, {size: 5}),
+    h6: spectacleComponent(Heading, {size: 6}),
+    img: Image,
+    li: ListItem,
+    p: Text,
+    strong: spectacleComponent(S, {type: "bold"}),
+    ul: List
+  }
+};
+
+
+export default class Markdown extends React.Component {
+  render() {
+    const content = typeof this.props.source !== "undefined" ?
```

Comment:
You've got a _different_ check for this same thing at https://github.com/FormidableLabs/spectacle/pull/87/files#diff-2d8f174e85e5217d3672c29687ab543cR12 -- maybe harmonize them?